app/views/application_templates/show.haml

```diff
@@ -1,169 +1,153 @@
+                  required: question.required?,
+                  class: 'form-control',
+                  id: index,
+                  value: @old_data[question.id] || ""
+              - when 'date'
+                = r.text_field question.unique_name,
+                  required: question.required?,
+                  class: 'form-control datepicker',
+                  placeholder: 'Click here to select date...',
+                  id: index,
+                  value: @old_data[question.id] || ""
+              - when 'number'
+                = r.number_field question.unique_name,
+                  required: question.required?,
+                  class: 'form-control',
+                  id: index,
+                  value: (@old_data[question.id] || "").to_i
+              - when 'yes/no'
+                = label_tag "#{index}_yes", 'Yes'
+                = r.radio_button question.unique_name, 'Yes',
+                  required: question.required?,
+                  id: "#{index}_yes",
+                  checked: @old_data[question.id] == 'Yes'
+                = label_tag "#{index}_no", 'No'
+                = r.radio_button question.unique_name, 'No',
+                  required: question.required?,
+                  id: "#{index}_no",
+                  checked: @old_data[question.id] == 'No'
+          = r.hidden_field question.unique_prompt_name,
+            value: question.prompt
+          = r.hidden_field question.unique_data_type_name,
+            value: question.data_type
+          = r.hidden_field question.id,
+            value: "#{question.id}-#{question.number}"
+    = hidden_field_tag :position_id, @template.position_id
 
-      - if @template.unavailability_enabled?
-        .heading Unavailability
-        .explanation
-          Please fill out your weekly unavailability for this semester via this calendar. This is
-          used to determine if you have enough available time to commit to this position.
-          Check off times when you are
-          %strong not
-          available to work.
-          Note that you can click and drag to check or uncheck a range of boxes.
-        %br
+    - if @template.unavailability_enabled?
+      %h2.heading Unavailability
+      .explanation
+        Please fill out your weekly unavailability for this semester via this calendar. This is
+        used to determine if you have enough available time to commit to this position.
+        Check off times when you are
+        %strong not
+        available to work.
+        Note that you can click and drag to check or uncheck a range of boxes.
+      .row.form-group
         = fields_for :unavailability do |u|
           =render partial: "application_drafts/unavailabilities_calendar",
                 locals: { form: u }
-
+    .row.form-group
       - if @template.eeo_enabled?
-        .heading Voluntary Equal Employment Opportunity Information
+        %h2.heading Voluntary Equal Employment Opportunity Information
```

Comment:
This heading being within a `.form-group` makes it not line up with the other headings on the page:

![2020-01-13_17-12](https://user-images.githubusercontent.com/240275/72296581-f4fbf980-3627-11ea-9906-9c44d199f582.png)